bin/sirius-script-rabpm-mngr.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+import argparse
+import subprocess
+
+
+def execute_ssh(hostname, *command):
+    subprocess.run(["ssh", f"lnls-bpm@{hostname}", *command])
```

Comment:
I assume that the ssh key is in ~/.ssh/id_ed25519_rabpm

Reply:
Do we need a special key name for this? (Don't know how such keys are managed in those machines.) It seems simpler to have its default name instead.

Reply:
Since the keys are generated with the specific function of accessing the BPM crates, I think that with this approach we leave the possibility of generating keys to other applications.